server/server.go

```diff
@@ -404,10 +393,40 @@ func (s *server) waitChAndUpdateClusterInfo(waitMsg string, readyCh chan struct{
 			}
 			s.clusterInfo.Subnets = s.network.subnets
 		}
+		s.log.Info(fmt.Sprintf("%s ready", msg))
```

Comment:
The same way we do with `getClusterInfo`, we could have a `setClusterInfo` which does the locking already, but this can be done in a follow up PR

Reply:
https://github.com/ava-labs/avalanche-network-runner/issues/430